bigframes/ml/preprocessing.py

```diff
@@ -305,8 +305,20 @@ def _compile_to_sql(
         array_split_points = {}
         if self.strategy == "uniform":
             for column in columns:
-                min_value = X[column].min()
-                max_value = X[column].max()
+                column_min = X[column].min()
+                column_max = X[column].max()
+
+                # Use Python value rather than Numpy value to serialization.
```

Comment:
This seems a bit fragile - should the sql generator be more strict in its formatting instead?

Reply:
+1 This is a bit scary and may indicate a bigger problem. Would adding numpy literal support to `BigQueryCompiler.visit_NonNullLiteral` allow us to avoid this?

Reply:
Moved changes to the ML SQL generation resided in `ml/sql.py`. Other data core APIs tests didn't have same complains so that `BigQueryCompiler.visit_NonNullLiteral` remain unaffected.